.changeset/heavy-poets-wink.md

```diff
@@ -0,0 +1,7 @@
+---
+'@apollo/server': patch
+---
+
+Errors reported by subgraphs (with no trace data in the response) are now accurately reflected in the numeric error stats.
```

Comment:
Does this changeset (and PR description) need to note that a particular version of Gateway is also needed to have the issue be actually fixed?

Reply:
Added in [e1b8122](https://github.com/apollographql/apollo-server/pull/7136/commits/e1b8122a78b3d400b347f612ef967cd179e05cb4) (with a FIXME to plug in final gateway version)